builtin/aws/ecs/platform.go

```diff
@@ -1350,13 +1388,13 @@ func (p *Platform) Destroy(
 
 type ALBConfig struct {
 	// Certificate ARN to attach to the load balancer
-	CertificateId string `hcl:"certificate"`
+	CertificateId string `hcl:"certificate,optional"`
```

Comment:
Should these be `*string` to tell unset versus empty string? 🤔

Reply:
I considered that, yeah, but opt'd to leave them as is because there are folks using `""` to mean not set atm so we'd have to check for `set but empty` anyway.